task/09-functions-n-closures-tasks.js

```diff
@@ -84,7 +90,10 @@ function getPolynom() {
  *   memoizer() => the same random number  (next run, returns the previous cached result)
  */
 function memoize(func) {
-    throw new Error('Not implemented');
+    let cache;
+    return () => {
+        return cache ? cache : cache = func();
```

Comment:
do not put variable assingment in the ternary operator. It's very easy to miss this when reading the code.